app.js

```diff
@@ -49,9 +49,16 @@ app.all("*", function (req, res) {
   return apiResponse.notFoundResponse(res, "Page not found");
 });
 
-app.use((err, req, res) => {
+// Error Handler
+app.use((err, req, res, next) => {
   if (err.name == "UnauthorizedError") {
     return apiResponse.unauthorizedResponse(res, err.message);
+  } else {
+    if (res.headersSent) {
```

Comment:
does this mean that if a response hasn't been sent, continue processing?

Reply:
Yes! If the response is sent we are allowing the express to handle it.
The next parameter is mandatory to receive the caught error. But the linting tool is not allowing without the use of the next parameter. In this way, we can satisfy the linting tool as well.